examples/oft-aptos/aptos.layerzero.config.ts

```diff
@@ -0,0 +1,161 @@
+}
+
+const bscSandboxContract: OmniPointHardhat = {
+    eid: EndpointId.BSC_V2_SANDBOX,
+    contractName: 'MyOFT',
+}
+
+const aptosSandboxContract: OmniPointHardhat = {
+    eid: EndpointId.APTOS_V2_SANDBOX,
+    contractName: 'oft',
+}
+
+const config: OAppOmniGraphHardhat = {
+    contracts: [
+        {
+            contract: ethereumContract,
+            config: ethConfig.accountConfig,
+        },
+        {
+            contract: bscContract,
+            config: bscConfig.accountConfig,
+        },
+        {
+            contract: ethereumSandboxContract,
+            config: ethConfig.accountConfig,
+        },
+        {
+            contract: bscSandboxContract,
+            config: bscConfig.accountConfig,
+        },
+        {
+            contract: aptosContract,
+            config: {
+                delegate: '58b730d07e98a22f2b357bee721115c986e4dc873c1884763708ee3d4006f74e',
+                owner: '58b730d07e98a22f2b357bee721115c986e4dc873c1884763708ee3d4006f74e',
+            },
+        },
+    ],
+    connections: [
+        {
+            from: aptosContract,
+            to: bscContract,
+            config: {
+                enforcedOptions: [
+                    {
+                        msgType: 1,
+                        optionType: ExecutorOptionType.LZ_RECEIVE,
+                        gas: 1000, // gas limit in wei for EndpointV2.lzReceive
+                        value: 0, // msg.value in wei for EndpointV2.lzReceive
+                    },
+                    {
+                        msgType: 2,
+                        optionType: ExecutorOptionType.LZ_RECEIVE,
+                        gas: 1000, // gas limit in wei for EndpointV2.lzCompose
+                        value: 0, // msg.value in wei for EndpointV2.lzCompose
+                    },
+                    // {
+                    //     msgType: 1,
+                    //     optionType: ExecutorOptionType.NATIVE_DROP,
+                    //     amount: 0, // amount of native gas token in wei to drop to receiver address
+                    //     receiver: '0x0000000000000000000000000000000000000000',
+                    // },
+                ],
+                sendLibrary: '0xbe533727aebe97132ec0a606d99e0ce137dbdf06286eb07d9e0f7154df1f3f10',
+                receiveLibraryConfig: {
+                    // Required Receive Library Address on Aptos
+                    receiveLibrary: '0xbe533727aebe97132ec0a606d99e0ce137dbdf06286eb07d9e0f7154df1f3f10',
+                    // Optional Grace Period for Switching Receive Library Address on Aptos
+                    gracePeriod: BigInt(0),
+                },
+                // Optional Receive Library Timeout for when the Old Receive Library Address will no longer be valid on Aptos
+                receiveLibraryTimeoutConfig: {
+                    lib: '0xbe533727aebe97132ec0a606d99e0ce137dbdf06286eb07d9e0f7154df1f3f10',
+                    expiry: BigInt(696969669),
```

Comment:
This number doesn't look accurate. We can't commit random data like this to devtools.

Reply:
yes we are going to set these to deafaults